file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -37,20 +50,105 @@ INTERFACE zif_aff_sajc_v1
       "! <p class="shorttext">Notification</p>
       "! Name of the class which contains the notification exit
       notification TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Delete</p>
+      "! Name of the class which contains the delete exit
+      delete       TYPE zif_aff_types_v1=>ty_object_name_30,
     END OF ty_exit_classes.
 
+  "! <p class="shorttext">Parameter Name</p>
+  "! Name of the parameter
+  TYPES ty_parameter_name TYPE c LENGTH 38.
```

Comment:
Do all values on DB have a length of 8 characters or less? If this was the case, I would set it to 8 chars. Otherwise I would keep it to the length stored on the database.

> The parameter name is set to 'read-only' in the configuration file, so they are only displayed, the customer cannot change them.

Keep in mind. You can avoid changing the length in a dedicated editor like in ADT. However, users can change it in the file (e.g., in a git repository when using abapGit).

Reply:
In a class-based catalog entry (these are the only ones which currently can be created and changed in the ADT) the length is restricted to 8 characters. But currently it is also possible to display catalog entry chains in the ADT (only display, no change). Here the length can be up to 38 characters. So, I think that I leave it to 38 characters.

It is no problem, if invalid entries are transferred via ABAPGit. The ADT only displays the parameters of the catalog entry which are defined in the ABAP class. If a parameter of the catalog entry is not defined in the ABAP class (e.g. because the class was changed meanwhile, or because some garbage is transferred via ABAPGit), the parameter is automatically removed from the parameter list (which is displayed in the ADT; to remove it from the database, the user must save and activate). An information message is displayed which informs the user about the removal and the reason (parameter does not exist in class).

Reply:
> In a class-based catalog entry (these are the only ones which currently can be created and changed in the ADT) the length is restricted to 8 characters. But currently it is also possible to display catalog entry chains in the ADT (only display, no change). Here the length can be up to 38 characters. So, I think that I leave it to 38 characters.

If there are values with 38 characters. We should stay with 38 characters.